main/EDMainMod.F90

```diff
@@ -276,15 +300,16 @@ subroutine ed_ecosystem_dynamics(currentSite, bc_in, bc_out)
        end if
     end if
 
+    ! SP has changes in leaf carbon but we don't expect them to be in balance. 
```

Comment:
At the moment, we can't run the host LM in SP mode because it thinks that FATES and SP mode are inconsistant (because FATES depends upon using 'use_cn' which is the opposite flag to 'use_sp'.  i got around this by using the code that allows some SP functionality -in- SP mode for the sake of n depositioin. However, we are currently making plans to break up the meaning of the 'use_cn' flag in CLM as it is an artifcact of past code and somewhat nonsensical under the present array of model configurations.   

That's all a bigger fish than this PR will deal with though. 

I am not, at least, triggering any balance check errors in the HLM. Maybe no carbon is getting through at all?